azavea/temperate
# Fix inline marketing page images in IE

## Overview 

Using absolute paths bypasess the AngularCLI webpack
configuration's attempt to optimize and inline, which
causes trouble in IE11.

I'm still not clear what the root cause is, but I believe
its related to a relative url being generated for the url()
CSS method that looks something like "city.ao34m21234mb0934"
which IE11 fails to recognize as a valid relative URL.

These are very large images (1mb and 630kb), it would be nice if we could
compress/optimize them further.

### Demo

![screen shot 2018-04-03 at 14 19 14](https://user-images.githubusercontent.com/1818302/38267918-d2bd0022-374a-11e8-9501-4b492b0e65e9.png)
![screen shot 2018-04-03 at 14 20 08](https://user-images.githubusercontent.com/1818302/38267925-d6a1e64e-374a-11e8-8883-9ffcdc8837d1.png)

### Notes

I was unable to reproduce the delayed image load in Edge on a real device, 2017 Surface Pro running latest Win10/Edge even with a cleared cache. Perhaps this was just an instance of slow image load of non-cached images? 

Looks like #1059 includes some other related issues, I'll leave it up to @azavea/temperate-design if they want to spin those out to a separate issue or address them as commits on this PR.

Some days I miss driving the California coast on Highway 1, that image is quite the throwback for me.

## Testing Instructions

- `./scripts/update && ./scripts/server --nginx` and view http://localhost:8000 in Edge and IE. All background images in both browsers on the marketing page should appear as intended.

Closes #1059
